doc/help/kdb.md

```diff
@@ -22,6 +22,7 @@ In this manual we give an overview of the tool suite
 `kdb`. It is part of Elektra’s tools. The tool suite `kdb` consists
 of individual commands. Most commands are independent and some commands
 are sharing an executable. Some commands are written as external scripts.
+Subcommands have to be the first after `kdb`
```

Comment:
Dot at end of sentence missing. The sentence is better placed after the first example (which does not show a command). Furthermore, the terminology is "commands" here, not subcommands (sorry that I used the wrong word before).